charts/alfresco-activiti/README.md

```diff
@@ -64,11 +64,12 @@ A Helm chart for Alfresco Activiti
 | ingress.hosts[0].paths[0].pathType | string | `"Prefix"` |  |
 | ingress.tls | list | `[]` |  |
 | license.secretName | string | `nil` |  |
-| livenessProbe.failureThreshold | int | `5` |  |
-| livenessProbe.initialDelaySeconds | int | `25` |  |
-| livenessProbe.path | string | `"/activiti-app/app/rest/locale"` |  |
-| livenessProbe.periodSeconds | int | `10` |  |
-| livenessProbe.timeoutSeconds | int | `5` |  |
+| livenessProbe.<<.failureThreshold | int | `5` |  |
+| livenessProbe.<<.initialDelaySeconds | int | `25` |  |
+| livenessProbe.<<.path | string | `"/activiti-app/actuator/health/readiness"` | Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0 |
+| livenessProbe.<<.periodSeconds | int | `10` |  |
+| livenessProbe.<<.timeoutSeconds | int | `5` |  |
```

Comment:
helm-docs doesn't seems to like that syntax, I prefer we just repeat those few keys

Reply:
mmmm Indeed, I missed that